### ISSUE TITLE
dart_services is failing to deploy on `main`

### ISSUE THREAD

**Issue by parlough** [dart-lang/dart-pad#2926]
I can't see why, but `dart_services` has been failing to deploy on the `main` channel for a while. [This run](https://github.com/dart-lang/dart-pad/runs/23324079558) is a recent failure.

**Comment by johnpryan** [comment]
The error seems to be with the build_storage_artifacts step:

```
Starting Step #1 - "Build storage artifacts"
Step #1 - "Build storage artifacts": Already have image (with digest): gcr.io/dart-services/flutter:main
Step #1 - "Build storage artifacts": Cannot open file, path = '.dart_tool/package_config.json' (OS Error: No such file or directory, errno = 2)
Finished Step #1 - "Build storage artifacts"
```

**Comment by johnpryan** [comment]
This is happening on the `stable` channel now, marking as a P0.

**Comment by johnpryan** [comment]
@devoncarew could you take a look?

**Comment by devoncarew** [comment]
I believe this is now fixed with c50da704bf18379cdcb39f225c4a09715c2ae420 and afc6013a08f872693f3b8bc1d2020c5d0b370e7f.

**Comment by johnpryan** [comment]
LGTM, thanks for investigating this.
